apps/backend/src/plugins/authorization/authorization.plugin.ts

```diff
@@ -76,9 +76,8 @@ const checkIsWhiteRoute = (url: string, whiteRoutes: string[]): boolean => {
 		return true;
 	}
 
-	const ROUTE_WITH_QUERY_INDEX = 0;
-
-	const routeWithoutQuery = route.split("?")[ROUTE_WITH_QUERY_INDEX] as string;
+	const URL_WITHOUT_QUERY_INDEX = 0;
+	const routeWithoutQuery = url.split("?")[URL_WITHOUT_QUERY_INDEX] as string;
```

Comment:
```suggestion
const [routeWithoutQuery] = url.split("?");
```

You can simplify this by using array destructuring instead of accessing the index with a constant. It improves readability and avoids unnecessary variable declarations.